docs/content/latest/contribute/core-database/build-from-src-macos.md

```diff
@@ -110,3 +110,12 @@ export PATH=$HOME/tools/apache-maven-3.6.3/bin:$PATH
 ```
 
 For building YugabyteDB Java code, you'll need to install Java and Apache Maven.
+
+## Build release package
+You can build a release package by executing:
+
+```shell
+$ ./yb_release
+......
+2020-10-27 13:55:40,856 [yb_release.py:283 INFO] Generated a package at '/Users/svarnau/code/yugabyte-db/build/yugabyte-2.5.1.0-6ab8013159fdca00ced7e6f5d2f98cacac6a536a-release-darwin-x86_64.tar.gz'```
```

Comment:
Is it common to have user names like 'svarnau' in the docs?

Reply:
I think it's ok. It's real example output.